### ISSUE TITLE
[Suggestion] Add an icon to indicate that a file is in queue

### ISSUE THREAD

**Issue by Martan404** [metherul/Automaton#48]
If you as a free user tries to download another file before the first one is ready then there's no indication if the file is queued or not. It would be really helpfull if this was shown

**Comment by metherul** [comment]
Good idea. I've added this to the to-do list for an upcoming release. Thanks for the suggestion!
